### PR TITLE
Icone added to music button in Music component for having a feedback …

### PR DESCRIPTION
**Pull request by TheoBoschetti**
…of the music playing

### CODE REVIEW

**Review by Gumatan — DISMISSED** [review]
c'est bon

File: frontend/public/index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
       href="https://fonts.googleapis.com/css?family=VT323&display=swap"
       rel="stylesheet"
     />
+    <link href="/your-path-to-fontawesome/css/all.css" rel="stylesheet" />
```
Comment (LoicBrassart) [inline frontend/public/index.html]: ? I'll need an explanation on this one

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Need some info

File: frontend/public/index.html (hunk 1 above)
Comment (TheoBoschetti) [reply frontend/public/index.html]: This is the link for using the icon from the music button

Comment (LoicBrassart) [reply frontend/public/index.html]: `your-path-to-fontawesome` ?

**Pull request merged**